app/Http/Controllers/ClassController.php

```diff
@@ -31,32 +33,37 @@ public function create()
      */
     public function store(Request $request)
     {
-        //
-        $data=[
-            'class_name'=>$request->class_name,
-            'capacity'=>$request->capacity,
-            'price'=>$request->price,
-            'timeFrom'=>$request->timeFrom,
-            'timeTo'=>$request->timeTo,
-            'is_fulled'=>isset($request->is_fulled),
+        $data=$request->validate([
+               
+            'class_name'=>'required|string',
+            'capacity'=>'required|string|max:50',
```

Comment:
capacity is not a string it is integer number `number of students class can take`